frontend/packages/console-dynamic-plugin-sdk/src/utils/common.ts

```diff
@@ -0,0 +1,13 @@
+export type ExtensionCommonK8sResource = {
```

Comment:
@vojtechszocs @christianvogt @spadgett can you approve this common type

Reply:
Is there a reason we wouldn't use `K8sResourceCommon` ?
Eventually some of these types will make it into the SDK.

Reply:
At least make a comment as to why this exists. It should be in the form of a `TODO` and that we will want to remove it in the future.